### ISSUE TITLE
Error when attempting to migrate from TSLint to ESLint

### ISSUE THREAD

**Issue by gs-scooter** [angular-eslint/angular-eslint#573]
I have an Angular project recently updated to v12 which is using TSLint. As per the documentation, I successfully ran the following command:
`ng add @angular/eslint-schematics`

This updated package.json as expected. I followed this up with the next command:
`ng g @angular-eslint/schematics:convert-tslint-to-eslint`

This produces the following output:
```
? Would you like to remove TSLint and its related config if there are no TSLint projects remaining after this conversion? Yes
? Would you like to ignore the existing TSLint config? Recommended if the TSLint config has not been altered much as it makes the new ESLint config cleaner. No
    
    INFO: We are now installing the "tslint-to-eslint-config" package into a tmp directory to aid with the conversion
    
    This may take a minute or two...
    

There was a critical error when trying to inspect your tslint.json: 

/bin/sh: npx: command not found
```
Is there a way to get more detailed info as to what's failing?

**Comment by JamesHenry** [comment]
@gs-scooter Do you not have npm installed on your machine?

What happens if you run:

```sh
node --version
```

and 

```sh
npm --version
```

?

**Comment by gs-scooter** [comment]
Node version: 12.22.1
NPM version: 6.14.12

I am able to successfully run other scripts with npx so I'm guessing it's a pathing issue. I did a little more digging around and confirmed that the error occurs when choosing _*not*_ to ignore the existing TSLint config. If I run the schematic and ignore the existing config, then it executes without error.

**Comment by JamesHenry** [comment]
Thanks @gs-scooter I was mightily confused suddenly a few folks were all reporting installation related issues but fortunately it's been narrowed down to the recent release of tslint-to-eslint-config 2.7.0.

So I am going to close this in favour of the previous issue we have on that. A workaround for now seems to be explicitly install tslint-to-eslint-config yourself 2.6.0 - you can see what other users are doing on this thread:

https://github.com/angular-eslint/angular-eslint/issues/571
